README.md

```diff
@@ -1,77 +1,138 @@
+         csi:
+           readOnly: true
+           driver: csi.sharedresource.openshift.io
+           volumeAttributes:
+             sharedConfigMap: share-default-config
+   ```
+
+See also:
+
+- [Simple example](docs/simple-example.md)
+- [Tekton example](docs/tekton-example.md)
+- [OpenShift BuildConfig example](docs/build-with-rhel-entitlements.md)
 
-For running on a 4.10 cluster which is *NOT* a Tech Preview cluster, you must employ the methodology described in the [Local Devlopment](#local-development)
-section.
-
-Once installed, the permission semantics around sharing resources is the next concern to consider.  In summary:
-- the `ServiceAccount` associated with a `Pod` needs access to the 'use' verb on the `SharedConfigMap` or `SharedSecret` referenced by any `CSIVolume`
-specified in a `Pod` that uses this repository's CSI Driver.
-- separately, any `User` can discover cluster scoped `SharedResources` based on the 'get' or 'list' permissions granted to them by their cluster
-or namespace administrator.
-
-The full definition of the `SharedConfigMap` can be found [here](deploy/0000_10_sharedconfigmap.crd.yaml) or `SharedSecret` custom resource can be found [here](deploy/0000_10_sharedsecret.crd.yaml).
-
-Under the [examples directory](examples) there is both a simple example of using a `SharedConfigMap` in a `Pod`, and an
-example of performing an OpenShift `Build` from a `BuildConfig` that uses Red Hat Entitlements available from your Red Hat Subscription when
-your subscription credentials are stored on your cluster via OpenShift Insights Operator, and resulting `Secret` for those
-credentials is made available to additional `Namespaces` via a `SharedSecret`.  Instructions for the `Pod` example are
-[here](docs/simple-example.md), and for the `BuildConfig` example are [here](docs/build-with-rhel-entitlements.md).
+## Features
 
-Next, for some details around support for updating `SharedConfigMap` or `SharedSecret` volumes as their corresponding Secrets or ConfigMaps change,
-please visit [here](docs/content-update-details.md).
+- ServiceAccounts must have the `use` permission to mount the respective
+  `SharedSecret` or `SharedConfigMap`. Volumes fail to mount otherwise - see 
+  [FAQ](docs/faq.md) for more details.
+- Automatic sync of the shared resource data (Secret/ConfigMap) into the mounting
+  `Pod`.
+- Automatic removal/restoration of shared resource data if the Pod's RBAC
+  permissions change at runtime.
+- Automatic removal/restoration of shared resource data if the backing
+  Secret/ConfigMap is deleted/re-created.
+- Survival of shared resource data with CSI driver restarts/upgrades.
+- Multiple `SharedSecret`/`SharedConfig` volumes within a `Pod`. Also supports
+  nested volume mounts within a container.
+
+The following CSI interfaces are implemented:
+
+- **Identity Service**: GetPluginInfo, GetPluginCapabilities, Probe
+- **Node Service**: NodeGetInfo, NodeGetCapabilities, NodePublishVolume, NodeUnpublishVolume
```

Comment:
highlight this is a partial implementation of the NodeService interface

Reply:
similarly, did you miss this, or not agree with citing the methods that return unimplemented @adambkaplan ?

Reply:
I think just leaving as-is is fine for now. I was trying to emulate what other kubernetes-sigs CSI drivers published in their READMEs.

Reply:
sounds good thanks for confirming